spring-boot-admin-server/src/test/java/de/codecentric/boot/admin/server/AbstractAdminApplicationTest.java

```diff
@@ -62,7 +62,7 @@ public void lifecycle() {
 					listInstances();
 					deregisterInstance(location.get());
 				}).assertNext((event) -> assertThat(event.opt("type")).isEqualTo("DEREGISTERED"))
-				.then(this::listEmptyInstances).thenCancel().verify(Duration.ofSeconds(60));
+				.then(this::listEmptyInstances).thenCancel().verify(Duration.ofSeconds(120));
```

Comment:
Do you know why it takes so much time in github build? Can we make it faster instead of waiting longer?

Reply:
First I want to have a passing, non flaky build, then, check why it is so slow :)